nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestDistributionObject.java

```diff
@@ -68,9 +72,25 @@ public void testDistribution() throws IOException {
         try (CloseableClientResponse response = get()) {
             assertEquals(200, response.getStatus());
             JsonNode node = mapper.readTree(response.getEntityInputStream());
-            JsonNode foo = node.get("foo");
-            assertNotNull(foo);
-            assertEquals("bar", foo.asText());
+            JsonNode key = node.get(Environment.PRODUCT_NAME);
+            assertNotNull(key);
```

Comment:
You can even assert the value by setting the Framework properties at the beginning of the test and removing them in a `finally` block.

Reply:
I couldn't delete properties I set up, so I replaced them by PlatformVersionHelper.UNKNOWN